tox.ini

```diff
@@ -42,11 +42,22 @@ commands =
 
 [testenv:integrationtests]
 basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH SOLC_BINARY BRANCH INTEGRATION_TESTS_REPO_URL
 deps =
 	-rrequirements/tests.txt
+setenv =
+	INT_TESTS_REPO_SSH_URL = git@github.com:gridsingularity/gsy-backend-integration-tests.git
+whitelist_externals =
+	git
+	bash
+commands_pre =
+	pip install -e .
+	git clone {env:INTEGRATION_TESTS_REPO_URL:{env:INT_TESTS_REPO_SSH_URL:}} {envtmpdir}/gsy-backend-integration-tests
 commands =
-    pip install -e .
-	behave --tags=-disabled integration_tests {posargs}
+	bash -c 'cp -r {envtmpdir}/gsy-backend-integration-tests/gsy_e_tests/integration_tests/ ./integration_tests/'
```

Comment:
Could you please explain why we now need to use `bash -c`? Doesn't `cp -r ...` work directly?

Reply:
As I explained in a previous comment, I encountered myself with the `bash -c 'for f in ./integration_tests/*.feature; do behave --tags=-slow --tags=-disabled "$f" || exit 1; done'` command, and therefore this `bash` command was whitelisted. 
Originally I thought of using `cp -r ...` directly, but then I noticed it could be also achieved by using `bash`, and would thus only be needed to whitelist `bash` only instead of `cp`, `bash`, and probably other commands.

Reply:
I see...but I suppose that whitelisting commands is done to actually keep the allowed commands under control, to avoid allowing them all. It might just be a matter of style: I would personally find it cleaner to just add `cp` to the whitelist and avoid the `bash -c '...'` part. But feel free to keep it as you like it more.